lib/socketCommands.js

```diff
@@ -293,8 +299,8 @@ class SocketCommands {
                     this.logEnabled = true;
                     this.adapter.requireLog(true, options);
                 }
-            } else if (type === 'fileChange') {
-                this.adapter.subscribeForeignFiles && this.adapter.subscribeForeignFiles(pattern, patternFile, options);
+            } else if (type === 'fileChange' && this.adapter.subscribeForeignFiles) {
```

Comment:
nit ... typeof === function?

Reply:
just moved the line above, tbh either its undefined or a function if we are not doing complete garbage in controller :smile: